Solutions/GoogleCloudPlatformIAM/Data/Solution_GoogleCloudPlatformIAM.json

```diff
@@ -0,0 +1,45 @@
+{
+    "Name": "GoogleCloudPlatformIAM",
+    "Author": "Microsoft - support@microsoft.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/google_logo.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The Google Cloud Platform Identity and Access Management (IAM) data connector provides the capability to ingest [GCP IAM logs](https://cloud.google.com/iam/docs/audit-logging) into Azure Sentinel using the GCP Logging API. Refer to [GCP Logging API documentation](https://cloud.google.com/logging/docs/api) for more information.\r\n  \r\n  **Underlying Microsoft Technologies used:** \r\n\r\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:\r\n\n  a. [Azure Monitor HTTP Data Collector API](https://docs.microsoft.com/azure/azure-monitor/logs/data-collector-api)\r\n\n \r\n\n  b. [Azure Functions](https://azure.microsoft.com/services/functions/#overview)\r\n\n",
+    "Data Connectors": [
+      "Data Connectors/GCP_IAM_API_FunctionApp.json"
+    ],
+    "Parsers": [
+      "Parsers/GCP_IAM.txt"
+    ],
+    "Workbooks": [
+      "Workbooks/GCP_IAM.json"
+    ],
+	"Analytic Rules": [
+      "Analytic Rules/GCPIAMDisableDataAccessLogging.yaml",
+	  "Analytic Rules/GCPIAMEmptyUA.yaml",
+	  "Analytic Rules/GCPIAMHighPrivilegedRoleAdded.yaml",
+	  "Analytic Rules/GCPIAMNewAuthenticationToken.yaml",
+	  "Analytic Rules/GCPIAMNewServiceAccount.yaml",
+	  "Analytic Rules/GCPIAMNewServiceAccountKey.yaml",
+	  "Analytic Rules/GCPIAMPrivilegesEnumeration.yaml",
+	  "Analytic Rules/GCPIAMPublicBucket.yaml",
+	  "Analytic Rules/GCPIAMServiceAccountEnumeration.yaml",
+	  "Analytic Rules/GCPIAMServiceAccountKeysEnumeration.yaml"
+    ],
+	"Hunting Queries": [
+      "Hunting Queries/GCPIAMChangedRoles.yaml",
+	  "Hunting Queries/GCPIAMDeletedServiceAccounts.yaml",
+	  "Hunting Queries/GCPIAMDisabledServiceAccounts.yaml",
+	  "Hunting Queries/GCPIAMNewCustomRoles.yaml",
+	  "Hunting Queries/GCPIAMNewServiceAccounts.yaml",
+	  "Hunting Queries/GCPIAMNewServiceAccountsKeys.yaml",
+	  "Hunting Queries/GCPIAMRareActionUser.yaml",
+	  "Hunting Queries/GCPIAMRareUA.yaml",
+	  "Hunting Queries/GCPIAMTopServiceAccountsFailedActions.yaml",
+	  "Hunting Queries/GCPIAMTopSrcIpAddrFailedActions.yaml"
+    ],
+    "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\GoogleCloudPlatformIAM",
+	"Version": "2.0.0",
+	"Metadata": "SolutionMetadata.json",
+	"TemplateSpec": true,
+	"Is1Pconnector": false
+  }
+  
```

Comment:
Hey @v-mchatla , please remove blank row

Reply:
As suggested, removed the extra space